### ISSUE TITLE
dilated convolution does not convergence with data format NHWC

### ISSUE THREAD

**Issue by weixsong** [tensorflow/tensorflow#27026]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): CentOS Linux release 7.4.1708
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): conda installation
- TensorFlow version (use command below): 1.9.0
- Python version: 3.6.5
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: CUDA Version 9.0.176， 
- GPU model and memory: V100, 16G

**Describe the current behavior**
I implemented WaveGlow model, in the model it contains dilated convolution, so in early time I used tf.nn.conv2d to implement the dilated convolution model- -- WaveNet, I used the default data format NHWC, but after many experiments, I found the model does not convergence. Even after 700K steps with a small learning rate, it did not convergence. I have tried many hyper-parameters settings.

So I changed the dilated convolution to a native implementation from [tensorflow-wavenet](https://github.com/ibab/tensorflow-wavenet). Then the WaveGlow model convergence quickly after 50k-80k steps.

Then, I want to understand what's wrong with my usage of tf.nn.conv2d, but no clue. Then I tried data format NCHW, the model convergences quickly after 70K steps. 

So I doubt there may be a bug in Tensorflow's implementation of dilated convolution with data format NHWC.

**Describe the expected behavior**
tf.nn.conv2d with data format NHWC should convergence as expected, it should behave the same as data format NCHW or naive dilated convolution implementation.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

my code: https://github.com/weixsong/WaveGlow
master branch is the naive implementation of dilated conv1d

model samples are also in the repo.

branch **tf_dilated_conv** is the implementation by tf.nn.conv2d with data format NHWC : https://github.com/weixsong/WaveGlow/tree/tf_dilated_conv

branch **tf_dilated_conv_channel_first** is the implementation by tf.nn.conv2d with data format NCHW  , https://github.com/weixsong/WaveGlow/tree/tf_dilated_conv_channel_first

**Other info / logs**
N/A

**Comment by smit-hinsu** [comment]
Could you provide us with a MCVE as described in https://stackoverflow.com/help/mcve that involves only a single TensorFlow operation and the results have large difference with data format NHWC and NCHW with a particular input?

Also, could you give it a try after setting 'TF_CUDNN_USE_AUTOTUNE' to '0' with NHWC data format to see if that helps? Winograd convolution that is used in case of NHWC is known to have higher numerical error. See section 4 in https://openreview.net/pdf?id=H1ZaRZVKg

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
